TrustSDK/Classes/Client/Types/Transaction.swift

```diff
@@ -0,0 +1,102 @@
+// Copyright Trust Wallet. All rights reserved.
+//
+// This file is part of TrustSDK. The full TrustSDK copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import Foundation
+
+public extension TrustSDK {
+    struct Transaction: Equatable {
+        public enum Action: String, Equatable {
+            case transfer
+        }
+
+        public let to: String
+        public let asset: UniversalAssetID
+        public let amount: String
+        public let action: Action
+        public let confirm: ConfirmType
+        public let from: String?
+        public let nonce: UInt64?
+        public let feeLimit: String?
```

Comment:
should we use `BigInt` for amount, fee and limit?

Reply:
and maybe nonce even

Reply:
yes, `BigInt` for `amount`, `fee`, `limit`, `UInt64` is enough for `nonce`